runHarness/WorkloadDrivers/AuctionWorkloadDriver.pm

```diff
@@ -1174,19 +1186,87 @@ sub startRun {
 	my $endRunStatus = "";
 	my $endRunStatusRaw = "";
 	$url = $self->getControllerURL() . "/run/$runName/state";
+	my @curIntervalNames;
+	my @lastIntervalNames;
+	my @statsRunning;
+
 	while (!$runCompleted) {
```

Comment:
One more thing that I just saw in testing with multiple appInstances.  Running with 4 appInstances, one of the appInstances hadn't started on the first fetch of the state, and so I saw:
```
Mon Jun 10 13:33:30 2019:    [appInstance1] Start: initial ramp at 200 users, duration:60s.
Mon Jun 10 13:33:30 2019:    [appInstance2] Start: initial ramp at 200 users, duration:60s.
Mon Jun 10 13:33:30 2019:    [appInstance3] Start: initial ramp at 200 users, duration:60s.
Mon Jun 10 13:33:30 2019:    [appInstance4] Start: initial ramp at 0 users, duration:60s.
```

Can you please add a sleep (say 30 seconds) before entering this loop.